charts/osm-arc/templates/metrics-agent-deployment.yaml

```diff
@@ -1,3 +1,54 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
This name should include a prefix for metrics agent since this security context will only be used for that deployment